src/nimblepkg/nimscriptapi.nim

```diff
@@ -106,6 +105,11 @@ proc printPkgInfo(): string =
   if backend.len == 0:
     backend = "c"
 
+  # Forward `namedBin` entries in `bin`
+  for k, v in namedBin:
+    if k notin bin:
+      bin.add k & "=" & v
```

Comment:
This still sucks and means you need to have code for parsing this new format (plus I'm sure this will have edge cases which will need to be covered in the future...)

Why can't namedBin be the source of truth?

Reply:
All said and done, we have to send the data from nimscript to nimble in an ini format.

https://github.com/nim-lang/nimble/blob/master/src/nimblepkg/nimscriptapi.nim#L101

No matter what, we will need some `a = b` equivalent formatting to pull this off.